src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -1713,7 +1713,30 @@ public final class JavadocTokenTypes {
     /** Col tag name. */
     public static final int COL_HTML_TAG_NAME = JavadocParser.COL_HTML_TAG_NAME;
 
-    /** Frame tag name. */
+    /**
+     * Frame tag html.
+     *
+     * <p><b>Example:</b></p>
+     * <pre>{@code <frame src="frame_a.html"> - Test for FRAME_HTML_TAG_NAME}</pre>
```

Comment:
Remove weird text after tag.
Make it:
`<frame name="top" src="./frame.html" />`

https://www.geeksforgeeks.org/html-frame-tag/